### ISSUE TITLE
Need english grammars ?

### ISSUE THREAD

**Issue by no-execution** [danijel3/SparrowhawkTest#5]
Thanks for your tutorial!

I am looking for an english grammar to do my text normalization tasks. But only a toy grammar provided by sparrowhawk toy demo.

And after one week search for grammar, I still cant find a better english grammar.

Do you have english grammar resources? Thanks!!!!!!

**Comment by danijel3** [comment]
Nope, don't have any. From what I've seen, there are various grammars lying around on the internet, but they are all tutorials and samples. Real "production-level" grammars are probably hidden from the public. If you know of a project that is developing an opensource version, please let me know - I'd gladly contribute.
